test/1_test_fullpath.js

```diff
@@ -1401,20 +1397,16 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
       const transaction = await web3.eth.getTransaction(complainTx.tx);
       const transactionBlock = await web3.eth.getBlock(transaction.blockNumber);
-      assert.isTrue(
-        voucherStatus.cancelFaultPeriodStart.eq(
-          new BN(transactionBlock.timestamp)
-        )
-      );
+      assert.isTrue(voucherStatus[4].eq(new BN(transactionBlock.timestamp)));
 
       // [1010.1000] = hex"A8" = 168 = REFUND_COMPLAIN
       assert.equal(
-        web3.utils.toHex(voucherStatus.status),
+        web3.utils.toHex(voucherStatus[0]),
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)